file-formats/cota/cota/type/zif_aff_cota_v1.intf.abap

```diff
@@ -0,0 +1,92 @@
+INTERFACE zif_aff_cota_v1
+  PUBLIC.
+
+
+  TYPES:
+    "! <p class="shorttext">Configuration</p>
+    "! Configuration
+    "! $required
+    BEGIN OF ty_configuration,
+      "! <p class="shorttext">Connection Target Name</p>
+      "! Connection target name
+      "! $required
+      conn_id                     TYPE c LENGTH 30,
+      "! <p class="shorttext">Connection Target Class</p>
+      "! Connection target class
+      "! $required
+      connection_target_class     TYPE c LENGTH 30,
+      "! <p class="shorttext">Multiple Application Destinations Allowed</p>
+      "! Multiple application destinations allowed
+      allow_multiple_destinations TYPE abap_bool,
```

Comment:
Mismatch "shorttext" vs. field name?

Reply:
Suggestion: 
Short text: Allow Multiple Application Destinations
Field Name: allow_multiple_destinations

Reply:
Sounds good to me...